src/todolist.js

```diff
@@ -16,11 +16,13 @@ const addItem = (task) => {
 };
 
 const removeItem = (index) => {
-  todoList.splice(index, 1);
+  const adjustedIndex = index - 1; // subtract 1 from the index
+  todoList.splice(adjustedIndex, 1);
+
   todoList.forEach((item, i) => {
     item.index = i + 1;
   });
```

Comment:
- [ ] Great effort working on the requested changes, But i notice the main functionality of this milestone is not met yet. If i delete the first task on the list the last task is the one that got deleted while the first task i was trying to delete still remain in the list,. Kindly try to fix this and be sure the particular task deleted is the one deleting from the list.

Below is a video to my claim. 👇 


https://github.com/Jessekay/ToDo-LIST/assets/78634841/4d89b59d-0364-4576-a41c-d662dd38d72d